locales/base/translation.json

```diff
@@ -2312,5 +2312,21 @@
         "title": "More coming soon!"
       }
     }
+  },
+  "earnFlow": {
```

Comment:
this (and a few other files) conflicts with main. I used `earnStablecoin` as the key, but I like `earnFlow` and updated it here https://github.com/valora-inc/wallet/pull/5394/files#diff-d3d67f4b3f8dbec345426f534268e40d9b087dfa5d83750165d63caf136cb24cR2329